### PR TITLE
Add Kotlin style settings to SquareAndroid

### PR DESCRIPTION
**Pull request by loganj**

### CODE REVIEW

File: configs/codestyles/SquareAndroid.xml [hunk 1]
```diff
@@ -75,6 +75,14 @@
   <JavaCodeStyleSettings>
     <option name="CLASS_NAMES_IN_JAVADOC" value="3" />
   </JavaCodeStyleSettings>
+  <JetCodeStyleSettings>
```
Comment (loganj) [inline configs/codestyles/SquareAndroid.xml]: I have no idea. This is what AS kicked out.

File: configs/codestyles/SquareAndroid.xml [hunk 2]
```diff
@@ -212,6 +220,14 @@
       <option name="TAB_SIZE" value="2" />
     </indentOptions>
   </codeStyleSettings>
+  <codeStyleSettings language="kotlin">
```
Comment (loganj) [inline configs/codestyles/SquareAndroid.xml]: Using some of the same settings as Java.

File: configs/codestyles/SquareAndroid.xml (hunk 1 above)
Comment (JakeWharton) [inline configs/codestyles/SquareAndroid.xml]: Jet is what Kotlin was originally named

**Comment by edenman** [comment]
:shipit:

**Comment by JakeWharton** [comment]
Thank you for contributing to the Square™ Java™ Code™ Styles™!

**Pull request merged**